### PR TITLE
Add link to HOWTO guide in README

### PR DESCRIPTION
**Pull request by Bharat123rox**
- Add link to HOWTO guides
- Add links to some core optimizers/layers
- Add an FAQ section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +225,73 @@ When working with large-scale input data, it is important to create large enough
 
 TODO: Add an example for running on Google Cloud.
 
+## Frequently asked questions (FAQs)
+A small selection of FAQs:
+
+**Question: Is there something similar to `tf.keras.Sequential`?**
```
Comment (jheek) [inline README.md:231]: We don't want the README to be too long. Could you make the questions link to the FAQ instead of inlining the answer?

Comment (Bharat123rox) [reply README.md:231]: Okay will link to the questions in the actual FAQ

Comment (Bharat123rox) [reply README.md:231]: Actually @jheek there seems to be no easy way to directly link it to the questions in the actual FAQ, so can you suggest what questions I can remove to make the README shorter?

Comment (jheek) [reply README.md:231]: Sorry for the late reply, I lost track of this issue.

I think it is better tot keep the FAQ questions out of the README if they cannot be inlined.

**Comment by jheek** [comment]
Hi, @Bharat123rox this PR has been inactive for a while, do you still intent to update it?

**Comment by Bharat123rox** [comment]
Hi @jheek I'm still working on it, sorry for not committing here

File: README.md (hunk 1 above)
Comment (Bharat123rox) [reply README.md]: Updated: Removed questions, retain link to the original FAQ

**Comment by avital** [comment]
Sorry @Bharat123rox that we dropped the ball on this so long ago! In the meanwhile we've totally rehauled our documentation, so I'll close this for now but please do let us know how we can continue to improve our documentation!

**Pull request closed without merging**